gerrit/install.sls

```diff
@@ -66,12 +66,12 @@ install_{{ name }}_plugin:
 {% endfor %}
 
 gerrit_war:
-  cmd.run:
-    - name: wget -qO {{ settings.base_directory }}/{{ gerrit_war_file }} {{ settings.package.base_url }}/{{ gerrit_war_file }}
-    - cwd: {{ settings.base_directory }}
+  file.managed:
```

Comment:
I'm not sure I understand what you mean.

The `gerrit_war` state's functionality hasn't changed. It now uses file.managed (with the recently added skip_verify) instead of calling `wget` (which isn't available on FreeBSD), but it still only downloads the Gerrit WAR.
file.managed (with a HTTP source + skip_verify) should work on all platforms, shouldn't it?